src/neuroconv/nwbconverter.py

```diff
@@ -53,26 +54,23 @@ def get_source_schema(cls) -> dict:
             source_schema["properties"].update({interface_name: unroot_schema(data_interface.get_source_schema())})
         return source_schema
 
-    def get_conversion_options_schema(self) -> dict:
```

Comment:
@h-mayorquin This was inconvenient to locate since it was defined after class methods, before `__init__` and isn't something that gets called until before `add_to_nwbfile`

I think it was left here because it maybe used to have been a class method